deployment/terraform/resources/batch.tf

```diff
@@ -5,6 +5,13 @@ resource "azurerm_storage_account" "pctasks-batch" {
   account_tier                    = "Standard"
   account_replication_type        = "LRS"
   allow_nested_items_to_be_public = false
+
+  identity {
```

Comment:
Double check this... This was intended to go under the Batch account, not the storage account.

My initial attempts to just add the MI to the Batch Pools failed (tasks never started). I assumed that adding this is what finally got it working, but clearly not?

Reply:
Well, I've removed this and redeployed and it still works :/ Not sure why my initial attempts failed.